aiida/manage/tests/main.py

```diff
@@ -319,7 +319,9 @@ def create_aiida_db(self):
             self.create_db_cluster()
         self.postgres = Postgres(interactive=False, quiet=True, dbinfo=self.dbinfo)
         # note: not using postgres.create_dbuser_db_safe here since we don't want prompts
-        self.postgres.create_dbuser(self.profile_info['database_username'], self.profile_info['database_password'])
+        self.postgres.create_dbuser(
```

Comment:
please add a comment here explaining why we need to have a user with createdb privileges here
(while migration typically should not require createdb privileges)

Reply:
done